### PR TITLE
Update CLI to 1.0.0-preview2-1-003182, BuildTools to 1.0.27-prerelease-01205-03

### PR DESCRIPTION
**Pull request by dagood**
See https://github.com/dotnet/corefx/pull/14933

> This version of CLI has the fix for #14215 (Running build.sh fails on Linux kernel 4.6.x).
> 
> Because the new CLI uses shared framework 1.1.0 rather than 1.0.0, we need to use a new matching BuildTools version (dotnet/buildtools#1286).
> 
> I uploaded a fake bootstrap Alpine package to CLI blob storage with this version so it can still build. Tested in a docker container.

### CODE REVIEW

**Comment by dagood** [comment]
The CI errors look like they happened due to the buildtools upgrade. Looking into them. (The upgrade span is around 2 months.)

**Comment by naamunds** [comment]
LGTM, once the potential issues with the buildtools upgrade are addressed.

File: tests/dir.props [hunk 1]
```diff
@@ -33,6 +33,22 @@
...
+
+  <!-- Output paths -->
+  <PropertyGroup>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(RootBinDir)obj\</BaseIntermediateOutputPath>
+    <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(BaseIntermediateOutputPath)\$(BuildOS).$(BuildArch).$(BuildType)</IntermediateOutputPath>
+    <OutputPath Condition="'$(OutputPath)' == ''">$(BaseIntermediateOutputPath)\$(BuildOS).$(BuildArch).$(BuildType)</OutputPath>
+    <FinalOutputPath Condition="'$(FinalOutputPath)' == ''">$(BinDir)</FinalOutputPath>
+  </PropertyGroup>
+
+  <Import Condition="Exists('$(ToolsDir)BuildVersion.targets')" Project="$(ToolsDir)BuildVersion.targets" />
```
Comment (dagood) [inline tests/dir.props]: [BuildVersion.targets provides](https://github.com/dotnet/buildtools/blob/704cb636b15bd4da3c6d7602631f116ff282eb41/src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildVersion.targets#L6) a value for `BuildVersionFile`, which is required by [Versioning.targets](https://github.com/dotnet/buildtools/blob/704cb636b15bd4da3c6d7602631f116ff282eb41/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L352-L357).

To generate that default, I needed `BaseIntermediateOutputPath` from `/dir.props`, so I brought over the full `Output paths` property group and a single entry from `Common properties`.

**Comment by dagood** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test (Known issue: https://github.com/dotnet/coreclr/issues/6574)

**Comment by dagood** [comment]
CI was green, but I had to remove from this PR the commit that I merged in https://github.com/dotnet/coreclr/pull/8831.

**Pull request merged**